### PR TITLE
Top-level README.md: use absolute links

### PR DESCRIPTION
**Pull request by mahilleb-msft**
For the benefit of https://docs.microsoft.com/samples/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Please check [here](https://aka.ms/csspeech/whatsnew) for release notes and olde
 This repository hosts samples that help you to get started with several features of the SDK.
 In addition more complex scenarios are included to give you a head-start on using speech technology in your application.
 
-We tested the samples with the latest released version of the SDK (1.5.0) on Windows 10, Linux (Ubuntu 16.04, Ubuntu 18.04, Debian 9), Android devices (API 23: Android 6.0 Marshmallow or higher), and iOS 11.4 devices.
+We tested the samples with the latest released version of the SDK (1.6.0) on Windows 10, Linux (Ubuntu 16.04, Ubuntu 18.04, Debian 9), Android devices (API 23: Android 6.0 Marshmallow or higher), and iOS 11.4 devices.
```
Comment (wolfma61) [inline README.md]: >1.6.0) [](start = 67, length = 6)

nice ;)

File: README.md [hunk 2]
```diff
@@ -127,40 +127,40 @@ If you want to build these quickstarts from scratch, please follow the articles
...
+| [Extended speech recognition sample for iOS](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/objective-c/ios/README.md#example-app-with-various-samples)           | iOS      | Demonstrates speech recognition using streams etc.|
+| [C# UWP DialogServiceConnector sample for Windows](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/virtual-assistant/csharp-uwp/) | Windows | Demonstrates speech recognition through the DialogServiceConnector and receiving activity responses. |
+| [C# Unity sample for Windows or Android](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/unity/speechrecognizer/) | Windows, Android |  Demonstrates speech recognition, intent recognition, and translation for Unity |
+| [C# Unity SpeechBotConnector sample for Windows](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/virtual-assistant/csharp-unity/) | Windows, Android | Demonstrates speech recognition through the SpeechBotConnector and receiving activity responses. |
```
Comment (wolfma61) [inline README.md]: >(https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/virtual-assistant/csharp-unity/ [](start = 50, length = 115)

not working for me ?

File: README.md [hunk 3]
```diff
@@ -127,40 +127,40 @@ If you want to build these quickstarts from scratch, please follow the articles
...
+| [Speech recognition and translation sample using JavaScript and Node.js](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/js/node/)                 | Node.js  | Demonstrates speech recognition, intent recognition, and translation |
+| [Speech recognition sample for iOS using a connection object](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/objective-c/ios/README.md#connection-object-sample)  | iOS      | Demonstrates speech recognition |
+| [Extended speech recognition sample for iOS](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/objective-c/ios/README.md#example-app-with-various-samples)           | iOS      | Demonstrates speech recognition using streams etc.|
+| [C# UWP DialogServiceConnector sample for Windows](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/virtual-assistant/csharp-uwp/) | Windows | Demonstrates speech recognition through the DialogServiceConnector and receiving activity responses. |
```
Comment (wolfma61) [inline README.md]: >https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/virtual-assistant/csharp-uwp/ [](start = 53, length = 112)

need to double check this one too

Comment (wolfma61) [reply README.md]: samples\csharp\uwp\virtualassistant-uwp

?

---
In reply to: [310005274](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/327#discussion_r310005274) [](ancestors = 310005274)

File: README.md (hunk 2 above)
Comment (wolfma61) [reply README.md]: I can't find this one?


---
In reply to: [310005017](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/327#discussion_r310005017) [](ancestors = 310005017)

Comment (mahilleb-msft) [reply README.md]: https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/csharp/unity/VirtualAssistantPreview/

File: README.md (hunk 3 above)
Comment (mahilleb-msft) [reply README.md]: https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/csharp/uwp/virtualassistant-uwp/

Thanks!!

**Pull request merged**